functests/common.sh

```diff
@@ -150,6 +150,38 @@ function check_drives() {
     check_drives_state Ready
 }
 
+# usage: check_drive_state <drive> <state>
+function check_drive_removed() {
+    drive="$1"
+    state="$2"
+    removed="drive is removed"
+    check_drives_state drive state
+    "${DIRECT_CSI_CLIENT}" drives list --drives="${drive}" -o wide
+    if ! "${DIRECT_CSI_CLIENT}" drives list --drives="${drive}" | grep -q -e "${drive}.*${removed}"; then
+        echo "$ME: error: ${drive} not found in removed state"
+        return 1
+    fi
+}
+
+# usage: check_volume_exist <state>
+function check_volume_exist() {
```

Comment:
this check needs to be fixed. 

- volumes will not be removed when the drive is detached
- `kubectl directpv volumes ls` will only show published and staged volumes (you did not see any volumes here in listing because these volumes are unstaged when the pods are deleted)
- unpublished and unstaged drives will show up only with `--all` in `kubectl directpv volumes ls` command
- If a drive is lost, its respective volumes should have an error tag indicating `[drive lost]`

Reply:
Addressed